docs/getting-started/getting-started-with-galaxy.rst

```diff
@@ -28,13 +28,15 @@ Using Custom Galaxy in the Cloud with Terra, and AnVIL
 ------------------------------------------------------------------------
 Galaxy is also integrated into the Terra cloud workspace. This provides users the ability to readily create a custom Galaxy environment scaled to their needs or with additional tools installed. Additionally, the Terra workspace is able to access a rich data corpus hosted in the cloud, including many controlled access datasets, thus allowing users to conduct analyses with this data in a compliance-based environment.
 
-Unlike WDL and CWL, Galaxy workflows in the near term are created and modified from the Galaxy workflow editor (GUI), instead of a text editor.
-
 Create a basic Galaxy workflow
 ------------------------------
 
+Unlike WDL and CWL, Galaxy workflows are currently created and modified from the Galaxy workflow editor (GUI), instead of a text editor.
+
 If you would like to create and run your own workflow in Galaxy, here  is a tutorial for `Creating, Editing, Importing Galaxy Workflows <https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/workflow-editor/tutorial.html>`__ from the Galaxy Training Network.
 
+In addition, the Galaxy Intergalactic Workflow Commission (IWC) team has an excellent tutorial for creating best practices workflows for IWC `here <https://github.com/galaxyproject/iwc/blob/main/workflows/README.md>`__. Although somewhat specific to the IWC directory structure, it discusses how to add the correct metadata, generate tests, and lint your workflow. It also shows how you can generate a ``.dockstore.yml`` using the `Planemo <https://planemo.readthedocs.io/en/latest/index.html>`__ command-line utilities.
```

Comment:
Might be good to develop "Although somewhat specific to the IWC directory structure" to explain to the user why we've noted it.